app/views/registrants/_state_id_number_newui2020.html.erb

```diff
@@ -0,0 +1,29 @@
+<%- include_race_and_party ||= false %>
+<%- include_date_of_birth ||= false %>
+<%- id_size = include_race_and_party ? 35 : 20 %>
+<%- if include_race_and_party %>
+  <fieldset>
+    <legend><%= I18n.t('txt.registration.identification_header') %></legend>
+<%- end %>
+    <ul class="flat">
+      <li class=registrant-form__state-id-number__line'>
+        <% if @registrant.has_state_license %>
+        <h3><%= form.label :state_id_number %><span class='required'>*<span class='required--text' style='display:none;'><%= I18n.t('required') %></span></span></h3>
+        <% else %>
+        <h3><%= form.label :id_no_state_id_number %><span class='required'>*<span class='required--text' style='display:none;'><%= I18n.t('required') %></span></span></h3>
```

Comment:
This ends up being semantically incorrect an generates:
<label for="registrant_id_no_state_id_number"> instead of for="registrant_state_id_number"
I think you can do 
`form.label :state_id_nummber, t('key.for.id_no_state_id_number')`
https://apidock.com/rails/ActionView/Helpers/FormBuilder/label

Reply:
@cwisdo